### DISCUSSION TITLE
Scoop意外卸载的处理办法

### DISCUSSION THREAD

**Discussion by giscus** [mcthesw/sworld-blog#11] [Announcements]
# Scoop意外卸载的处理办法 - Sworld

这几天发现电脑上的Scoop莫名其妙消失了，我觉得可能是UniGetUI的问题，总之我的Scoop被卸载了，使用Scoop安装的软件的环境变量也失效了，这导致我写代码的相关环境无法使用。在重新安装Scoop后，环境变量没有恢复，而我发现路径C:\Users\用户名\scoop没有丢失，那么说明软件文件和持久化数据应该没有丢失，寻找一些资料后，我在一篇Github Issue中找到了解决办法，如果你

https://blog.sworld.club/2024/09/6a5cbb2e/

**Comment by mcthesw** [comment]
不得不说Scoop被卸载真的太危险了
